source/uwp/Visualizer/AdaptiveCardVisualizer.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>XamlCardVisualizer</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
```

Comment:
Don't change the minVersion unless it's required

Reply:
Same as this comment above: https://github.com/microsoft/AdaptiveCards/pull/7205#discussion_r848703347

I assume changing the version on the visualizer is more ok though right? Because it's an internal tool, really.